### DISCUSSION TITLE
It is possible to add more fields in translation? example tranlsation create at

### DISCUSSION THREAD

**Discussion by sefirosweb** [spatie/laravel-translatable#443] [Q&A] [answered]
Hello guys awsome package,

I have one dooubt, it is possible to add more fields in the specific translation? example i'm interested to know the create_at / update_at / other fields for specific translation

Maybe the json only havae lenguage and translation directly right? they dont have a objet to add more fields, example:
```json
{
  'en' : {
    'translation':'xxx'
    'field_1':'xx',
    'created_at':'xx'
  },
  'en' : {
    'translation':'yyy'
    'field_1':'yyy',
    'created_at':'yyy'
  }
}
```

**Comment by rsids** [comment, marked as answer]
Yes, this is possible, the data in my db looks like this:
```
{
  "body": {
    "en": "<p>content EN<\/p>",
    "de": "<p><\/p>",
    "nl": "<p><\/p>"
  },
  "text_image": {
    "en": {
      "title": "header title en",
      "content": "content en",
      "img": "/img.jpg"
    },
    "de": {
      "title": null,
      "content": ""
    },
    "nl": {
      "title": "header title nl",
      "content": "<p><\/p>",
      "img": "/img.jpg"
    }
  }
}
```
I can get the translated values with
`$page->getTranslation('content->text_image', $locale);`, which gives me the translated object. Fallback locale doesn't seem to work.

**Reply by sefirosweb** [reply]
Thanks I got it
